.bazelrc

```diff
@@ -37,6 +41,11 @@ build --noexperimental_check_output_files
 # Don't bother building targets which aren't dependencies of the tests.
 test --build_tests_only
 
+# Show all the problems
+test --keep_going
+test --test_output=errors
+build --verbose_failures
```

Comment:
I'm fine with `--test_output=errors` being here but the other two I don't see necessary by default (e.g. I use them on occasions, only).

Reply:
removed